plugins/azure-functions/src/components/OverviewTableComponent/OverviewTable.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Box, Card, Link, LinearProgress } from '@material-ui/core';
+import { FunctionsData } from '@backstage/plugin-azure-functions-common';
+import { Table, TableColumn } from '@backstage/core-components';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+
+type States = 'Waiting' | 'Running' | 'Paused' | 'Failed';
+
+const State = ({ value }: { value: States }) => {
+  const colorMap = {
+    Waiting: '#dcbc21',
+    Running: 'green',
+    Paused: 'black',
+    Failed: 'red',
+  };
+  return (
+    <Box display="flex" alignItems="center">
+      <span
+        style={{
+          display: 'block',
+          width: '8px',
+          height: '8px',
+          borderRadius: '50%',
+          backgroundColor: colorMap[value],
+          marginRight: '5px',
+        }}
+      />
+      {value}
+    </Box>
+  );
+};
+
+type FunctionTableProps = {
+  data: FunctionsData[];
+  loading: boolean;
+};
+
+const DEFAULT_COLUMNS: TableColumn<FunctionsData>[] = [
+  {
+    title: 'name',
+    highlight: true,
+    render: (func: FunctionsData) => {
+      return (
+        <Link href={func.href} target="_blank">
+          {func.functionName}
+        </Link>
+      );
+    },
+  },
+  {
+    title: 'location',
+    render: (func: FunctionsData) => func.location ?? 'unknown',
+  },
+  {
+    title: 'status',
+    render: (func: FunctionsData) => <State value={func.state as States} />,
+  },
+  {
+    title: 'last modified',
+    render: (func: FunctionsData) =>
+      new Date(func.lastModifiedDate).toUTCString(),
+  },
```

Comment:
If possible, it would be great if this lined up to this Architectural Decision Record here:
https://backstage.io/docs/architecture-decisions/adrs-adr012

That is, keeping the date formatting consistent to the user's locale, rather than showing in UTC and making the user need to interpret it?